tests/integration/__init__.py

```diff
@@ -30,11 +31,41 @@ def url(self, uri=""):
     def goto(self, uri=""):
         self.driver.get(self.url(uri))
 
-    def login(self, email, password, **kwargs):
+    def ia_login(self, email, password, test=None, **kwargs):
+        self.driver.get('https://archive.org/account/login.php')
```

Comment:
Can we parameterize these URL endpoints? I can imagine cases where we'd want to be able to specify e.g. other dev endpoints. (Same applies to all hard-coding of endpoints in these tests.)